### PR TITLE
Fix #2027

### LINKED ISSUE #2027: microk8s MOTD popup on SSH
When I SSH'd into my VM I got this:

Screenshot 2019-12-09 at 16 47 57

I'm sure microk8s is interesting, but it's not VVV, lets get this removed from MOTD

### PR DESCRIPTION
**Pull request by Mte90**
ref #2027 

This remove the motd of the screen, changes tested.

### CODE REVIEW

File: provision/provision.sh [hunk 1]
```diff
@@ -274,6 +274,9 @@ cleanup_terminal_splash() {
   if [[ -f /etc/update-motd.d/50-landscape-sysinfo ]]; then
     rm /etc/update-motd.d/50-landscape-sysinfo
   fi
+  if [[ -f /etc/update-motd.d/80-live-patch ]]; then
+    rm /etc/update-motd.d/80-live-patch
```
Comment (msaggiorato) [inline provision/provision.sh]: For my, without running this specifically i have the `80-livepatch` file. Maybe we should remove that aswell?

Comment (Mte90) [reply provision/provision.sh]: Well this command will remove that file

Comment (msaggiorato) [reply provision/provision.sh]: But the command here removes the file named 80-live-patch. I mentioned 80-livepatch.

Comment (Mte90) [reply provision/provision.sh]: you are right... I didn't saw that. fixed.

**Pull request merged**